src/main/java/interface_adapter/execute_buy/ExecuteBuyPresenter.java

```diff
@@ -14,23 +14,18 @@ public void prepareSuccessView(ExecuteBuyOutputData outputData) {
                         outputData.newPortfolio(),
                         outputData.newBalance()
                 )
-       );
+        );
     }
 
     @Override
-    public void prepareInsufficientFundsView() {
-
+    public void prepareInsufficientBalanceErrorView() {
     }
 
     @Override
-    public void prepareStockNotFoundView() {
-
+    public void prepareStockNotFoundErrorView() {
     }
 
     @Override
     public void prepareValidationErrorView() {
```

Comment:
Refer to my comment below. `Error` and `Exception` are two different concepts and should not be misused.